src/Schema/Blueprint.php

```diff
@@ -268,13 +268,22 @@ public function tokenList(string $column, TokenizerFunction $function, string $t
      */
     public function interleaveInParent(string $table): InterleaveDefinition
     {
-        $command = new InterleaveDefinition(
-            $this->addCommand('interleaveInParent', compact('table'))->getAttributes(),
-        );
-
-        $this->commands[count($this->commands) - 1] = $command;
+        return $this->interleaveIn($table, true);
+    }
 
-        return $command;
+    /**
+     * @param string $table
+     * @param bool $parent
+     * @return InterleaveDefinition
+     */
+    public function interleaveIn(string $table, bool $parent = false): InterleaveDefinition
+    {
+        return $this->commands[] = new InterleaveDefinition(
+            $this->createCommand('interleaveInParent', [
```

Comment:
How about changing the command name  to `interleaveIn`?

Reply:
Fixed.